### ISSUE TITLE
[DatePicker][Accessibility] Label isn't a real label

### ISSUE THREAD

**Issue by nantunes** [lumada-design/hv-uikit-react#1692]
• Form elements must have labels (https://dequeuniversity.com/rules/axe/3.5/label?application=axeAPI)

   (#root > div > div > div > input)

   <input class="HvDatePicker-input" placeholder="Custom placeholder" type="text" readonly="" tabindex="-1" value="">

Also, when providing a custom label, the rendered label isn't a real label and the button loses the accessible content.
 • This element has role of "button" but does not have a name available to an accessibility API. Valid names are: element content, aria-label undefined, aria-labelledby undefined.

   (#root > div > div > div)

   <div class="HvDatePicker-root HvDatePicker-inputCalendarClosed" role="button" tabindex="0"><input class="HvDatePicker-inpu...</div>
